### PR TITLE
Add json schema

### PR DESCRIPTION
**Pull request by chainlink**

### CODE REVIEW

File: pachyderm/values.yaml [hunk 1]
```diff
@@ -19,16 +19,16 @@ pachd:
     #  memory: 2G  
   logLevel: info
   metrics:
-    enabled: true
+    enabled: "true"
```
Comment (robert-uhl) [inline pachyderm/values.yaml:22]: Can we keep Boolean values as actual booleans in the values file, and turn them into strings if needed by quoting in the templates? That way conditionals are not surprising.

Comment (robert-uhl) [reply pachyderm/values.yaml:22]: Applies to the other instances of true/false throughout.

File: pachyderm/values.yaml [hunk 2]
```diff
@@ -109,8 +109,8 @@ imageCredentials: {}
 # move under ingress? How to make work with cert-manager, maybe add
 # tls option to pachctl create secret?
 tls:
-  crt:
-  key:
+  crt: ""
```
Comment (robert-uhl) [inline pachyderm/values.yaml:112]: How do you feel about `nil` for unspecified, versus empty string? Either can ultimately work, of course.

File: pachyderm/values.yaml [hunk 3]
```diff
@@ -19,16 +19,16 @@ pachd:
...
-  authenticationDisabledForTesting: false
-  exposeObjectAPI: false
+  blockCacheBytes: "1G"
+  authenticationDisabledForTesting: "false"
+  exposeObjectAPI: "false"
   clusterDeploymentID: ""
-  requireCriticalServersOnly: false
-  ppsWorkerGRPCPort: 80
+  requireCriticalServersOnly: "false"
+  ppsWorkerGRPCPort: "80"
```
Comment (robert-uhl) [inline pachyderm/values.yaml:31]: Can this be a number, rather than a string?

**Review by robert-uhl — COMMENTED** [review]
I feel that in general the types in `values.yaml` should be the actual types of values: numbers for integers and floats, Booleans for true/false values and so forth, and that they should be properly quoted at the point of use.

What do you think?

File: pachyderm/values.yaml (hunk 2 above)
Comment (chainlink) [reply pachyderm/values.yaml:112]: I don't have strong opinions one way or the other (they were about half and half, so added the remainder for consistency)

**Review by robert-uhl — APPROVED** [review]
Looks pretty good to me! I can live with empty strings for the credentials for the moment, no problem☺

**Pull request merged**